drivers/SmartThings/philips-hue/src/init.lua

```diff
@@ -694,15 +690,8 @@ local function init_bridge(driver, device)
     ))
     cosock.spawn(
       function()
-        local duration = 30 -- seconds
-        local timeout_time = cosock.socket.gettime() + duration
-        local time_remaining = duration
         local bridge_info, err
-
-        while time_remaining > 0 do
-          log.info_with({ hub_logs = true }, string.format(
-            "time remaining bridge init for %s" .. time_remaining, (device.label or device.id or "unknown device")
-          ))
+        while true do
           bridge_info, err = HueApi.get_bridge_info(ip)
```

Comment:
What exactly is involved here?  This is a network operation correct? Just want to make sure this isn't a super tight loop on the continue when bridge_info is nil

Reply:
Yeah this is an un-authed REST call so it doesn't take an API instance, it creates and then tears down a REST client under the hood.

Regardless of that, though, I do believe that we should probably have a (bounded) back-off here.